client/src/pages/private/CohortDetails.js

```diff
@@ -56,9 +45,8 @@ export default ({ match }) => {
     try {
       setIsLoading(true);
       const cohortData = await fetchCohort({ cohortId });
-      setCohort(cohortData);
-      const cohortParticipantsData = (await fetchCohortParticipants({ cohortId })) || [];
-      setRows(cohortParticipantsData);
+      setCohort(cohortData.cohort);
```

Comment:
Is there another check needed?
`cohortData?.cohort`

Reply:
`fetchCohort` throws an error so I don't think that's necessary